src/drivers/ms5611/ms5611.cpp

```diff
@@ -98,8 +98,12 @@ enum MS5611_BUS {
  * MS5611/MS5607 internal constants and data structures.
  */
 
-/* internal conversion time: 9.17 ms, so should not be read at rates higher than 100 Hz */
-#define MS5611_CONVERSION_INTERVAL	25000	/* microseconds */
+/*
+ * Maximum internal conversion time for OSR 1024 is 2.28 ms. We set an update
+ * rate of 100Hz which is be very safe not to read the ADC before the
+ * conversion finished
+ */
+#define MS5611_CONVERSION_INTERVAL	10000	/* microseconds */
```

Comment:
If it doesn't affect the internal read rate then the self heating issue does not apply and this is something that should be considered for all boards.